src/voice/connection.rs

```diff
@@ -267,6 +310,13 @@ impl Connection {
             self.udp.send_to(&bytes, self.destination)?;
         }
 
+        // Reconfigure encoder bitrate.
+        // From my testing, it seemed like this needed to be set every cycle.
+        // -- FelixMCFelix
```

Comment:
The signature's not necessary, someone can always `git blame` to find out who said this.